RockLib.DistributedTracing.AspNetCore/RockLib.DistributedTracing.AspNetCore.csproj

```diff
@@ -23,13 +23,17 @@
 	<PropertyGroup Condition="'$(Configuration)'=='Release'">
 		<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
 	</PropertyGroup>
+	<ItemGroup>
+	  <None Remove="System.Text.Encodings.Web" />
+	</ItemGroup>
 	<ItemGroup>
 		<None Include="..\LICENSE.md" Pack="true" PackagePath="" />
 		<None Include="..\icon.png" Pack="true" PackagePath="" />
 	</ItemGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
 		<PackageReference Include="OpenTelemetry" Version="1.3.1" />
+		<PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" />
```

Comment:
This was from the Http.Abstractions could you move it down to there?

Reply:
Moved the package reference location.